bin/add_celltypes_to_sce.R

```diff
@@ -331,7 +331,9 @@ if (has_singler && has_cellassign) {
   # read in consensus table
   consensus_ref_df <- readr::read_tsv(opt$consensus_celltype_ref) |>
     # select unique combinations of consensus refs based on ontology columns
-    dplyr::select(blueprint_ontology, panglao_ontology, consensus_ontology, consensus_annotation) |>
+    # TODO: Update when incorporating scimilarity
+    dplyr::select(blueprint_ontology, panglao_ontology, consensus_ontology = cellassign_singler_pair_ontology, consensus_annotation = cellassign_singler_pair_annotation) |>
```

Comment:
just noting the long line, but since this will be updated, not a big deal
```suggestion
    dplyr::select(
      blueprint_ontology,
      panglao_ontology,
      consensus_ontology = cellassign_singler_pair_ontology,
      consensus_annotation = cellassign_singler_pair_annotation
    ) |>
```